codex_of_the_damned/templates/archetypes/bibliodeque/419-operation.html

```diff
@@ -0,0 +1,48 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/_layout-erratum.html" as erratum with context %}
+{% import "archetypes/bibliodeque/419-operation.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-osebo.svg') }}"> 419 Operation</h1>
```

Comment:
No icon

Reply:
Icon to process : http://www.vekn.net/images/stories/icons/clans/iconclanosebo.gif